src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs

```diff
@@ -175,7 +175,13 @@ private bool StartNewModelComputation(CompletionService completionService, Compl
 
             this.sessionOpt = new Session(this, computation, Presenter.CreateSession(TextView, SubjectBuffer, null));
 
-            sessionOpt.ComputeModel(completionService, trigger, _roles, GetOptions());
+            var options = GetOptions();
+            if (_isDebugger)
+            {
+                options = options.WithChangedOption(EditorCompletionOptions.UseSuggestionMode, true);
```

Comment:
This starts the session with the builder on, but the user can turn it off with ctrl-alt-space. When the builder is driven by SuggestionModeCompletionProvider providing a builder (rather than the option), we don't respect the option, so you can't turn it off.